analyzer/src/funTest/assets/projects/synthetic/dotnet-expected-output.yml

```diff
@@ -16,205 +16,40 @@ project:
     path: "<REPLACE_PATH>"
   homepage_url: ""
   scopes:
-  - name: "net45"
-    dependencies:
-    - id: "NuGet::System.Threading.Tasks.Extensions:4.5.4"
-      dependencies:
-      - id: "NuGet::System.Collections:4.3.0"
-        dependencies:
-        - id: "NuGet::Microsoft.NETCore.Platforms:1.1.0"
-        - id: "NuGet::Microsoft.NETCore.Targets:1.1.0"
-        - id: "NuGet::System.Runtime:4.3.0"
-      - id: "NuGet::System.Runtime:4.3.0"
-      - id: "NuGet::System.Runtime.CompilerServices.Unsafe:4.5.3"
-        dependencies:
-        - id: "NuGet::System.Runtime:4.3.0"
-          dependencies:
-          - id: "NuGet::Microsoft.NETCore.Platforms:1.1.0"
-          - id: "NuGet::Microsoft.NETCore.Targets:1.1.0"
-      - id: "NuGet::System.Threading.Tasks:4.3.0"
-        dependencies:
-        - id: "NuGet::Microsoft.NETCore.Platforms:1.1.0"
-        - id: "NuGet::Microsoft.NETCore.Targets:1.1.0"
-        - id: "NuGet::System.Runtime:4.3.0"
-  - name: "netcoreapp3.1"
+  - name: "install"
     dependencies:
+    - id: "NuGet::Antlr:3.4.1.9004"
+    - id: "NuGet::Newtonsoft.Json:5.0.4"
     - id: "NuGet::System.Globalization:4.3.0"
-      dependencies:
-      - id: "NuGet::Microsoft.NETCore.Platforms:1.1.0"
-      - id: "NuGet::Microsoft.NETCore.Targets:1.1.0"
-      - id: "NuGet::System.Runtime:4.3.0"
+    - id: "NuGet::System.Runtime.CompilerServices.Unsafe:4.5.3"
     - id: "NuGet::System.Threading:4.0.11"
+    - id: "NuGet::System.Threading.Tasks.Extensions:4.5.4"
       dependencies:
-      - id: "NuGet::System.Runtime:4.1.0"
-        dependencies:
-        - id: "NuGet::Microsoft.NETCore.Platforms:1.0.1"
-        - id: "NuGet::Microsoft.NETCore.Targets:1.0.1"
-      - id: "NuGet::System.Threading.Tasks:4.0.11"
-        dependencies:
-        - id: "NuGet::Microsoft.NETCore.Platforms:1.0.1"
-        - id: "NuGet::Microsoft.NETCore.Targets:1.0.1"
-        - id: "NuGet::System.Runtime:4.1.0"
-  - name: "netcoreapp3.1-dev"
-    dependencies:
+      - id: "NuGet::System.Runtime.CompilerServices.Unsafe:4.5.3"
     - id: "NuGet::WebGrease:1.5.2"
       dependencies:
       - id: "NuGet::Antlr:3.4.1.9004"
       - id: "NuGet::Newtonsoft.Json:5.0.4"
+    - id: "NuGet::foobar:1.2.3"
 packages:
 - id: "NuGet::Antlr:3.4.1.9004"
   purl: "pkg:nuget/Antlr@3.4.1.9004"
   authors:
-  - "Terence Parr"
-  declared_licenses: []
-  declared_licenses_processed: {}
+  - "Terence Parr <>"
+  declared_licenses:
+  - ""
```

Comment:
We should not have an empty value here if empty, same for declared_licenses_processed below

Reply:
Done!